yarn.lock

```diff
@@ -7485,7 +7485,7 @@ querystringify@^2.1.1:
   resolved "https://registry.yarnpkg.com/querystringify/-/querystringify-2.1.1.tgz#60e5a5fd64a7f8bfa4d2ab2ed6fdf4c85bad154e"
   integrity sha512-w7fLxIRCRT7U8Qu53jQnJyPkYZIaR4n5151KMfcJlO/A9397Wxb1amJvROTK6TOnp7PfoAmg/qXiNHI+08jRfA==
 
-randombytes@^2.0.0, randombytes@^2.0.1, randombytes@^2.0.5:
+randombytes@^2.0.0, randombytes@^2.0.1, randombytes@^2.0.5, randombytes@^2.1.0:
```

Comment:
Wow, we rely on 4 versions of the same lib, 3 of which are compatible. That looks like it could use some dependency alignment 👀 

Don't bother with this, but I'll raise an internal issue to improve this.

Reply:
Raised in https://github.com/mswjs/msw/issues/516